src/vivarium_gates_child_iv_iron/model_specifications/iv_iron_child.yaml

```diff
@@ -53,8 +53,10 @@ components:
 configuration:
     input_data:
         input_draw_number: 0
-        artifact_path: '/ihme/costeffectiveness/artifacts/vivarium_gates_child_iv_iron/south_asia.hdf'
-        fertility_input_data_path: '/ihme/costeffectiveness/results/vivarium_gates_iv_iron/v8.2_child_output_only/south_asia/2022_06_16_12_36_52/child_data/'
+        # Artifact should be defined at runtime using -i flag (available in src/vivarium_gates_iv_iron/artifacts)
+        #artifact_path: '/ihme/costeffectiveness/artifacts/vivarium_gates_child_iv_iron/south_asia.hdf'
+        # Use child_data folder created by IV iron maternal model
+        #fertility_input_data_path: '/ihme/costeffectiveness/results/vivarium_gates_iv_iron/v8.2_child_output_only/south_asia/2022_06_16_12_36_52/child_data/'
     interpolation:
```

Comment:
You should not have anything like a cluster specific path in the archive. It's not useful, as the replication test *must* be done on some platform other than the cluster.

Reply:
Makes sense but I'm not sure what to put instead since that folder is the output of a separate model run so there's not really a "standard" or relative location for it. Maybe I could just put an empty string?

Reply:
Maybe put a FIXME tag with comments about what to put here.